client/lib/feed-stream-store/feed-stream.js

```diff
@@ -237,7 +234,7 @@ export default class FeedStream {
 		do {
 			const key = this.postKeys[ i ];
 			if ( ! key.isGap ) {
-				date = FeedPostStore.get( key )[ this.dateProperty ];
+				date = key[ this.dateProperty ];
```

Comment:
because the date is now being stored on the postKey directly. we just look at, and store, a different property based on the stream.